oak_functions/gcp/weather_lookup_api_config.yaml

```diff
@@ -0,0 +1,28 @@
+type: google.api.Service
+config_version: 3
+
+#
+# Name of the service configuration.
+#
+name: '*.apigateway.endpoints.oak-ci.cloud.goog'
+
+#
+# API title to appear in the user interface (Google Cloud Console).
+#
+title: Oak Functions gRPC API Gateway
+apis:
+  - name: oak.functions.server.GrpcHandler
+
+#
+# API usage restrictions.
+#
+usage:
+  rules:
+    # Invoke method can be called without an API Key for now.
+    - selector: oak.functions.server.GrpcHandler.Invoke
+      allow_unregistered_calls: true
+
+backend:
+  rules:
+    - selector: '*'
+      address: grpcs://oak-functions-weather-lookup-62sa4xcfia-nw.a.run.app
```

Comment:
Where do we get this from?

Reply:
Could this be made parametric, since this is the only part of this file that is specific to the weather_lookup app?

Reply:
Added a comment on where this comes from. gcloud does not support parameterising these config files.